web/src/stats/stats.ts

```diff
@@ -654,17 +744,22 @@ function populate_messages_sent_by_client(data) {
     $("#pie_messages_sent_by_client button").on("click", function () {
         if ($(this).attr("data-user")) {
             set_user_button($(this));
-            user_button = $(this).attr("data-user");
+            user_button = $(this).attr("data-user")! as "everyone" | "user";
```

Comment:
With the zod schema mentioned earlier, we can parse the value without compromising type safety.